Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/DeviceFeatures.h

```diff
@@ -77,6 +77,9 @@ namespace AZ
 
             //! Whether PipelineLibrary related serialized data needs to be loaded/saved explicitly as drivers (like dx12/vk) do not support it internally
             bool m_isPsoCacheFileOperationsNeeded = true;
+
+            //! Whether supports undefined swizzle tile resource
+            bool m_tiledResource = true;
```

Comment:
Does Metal and Vulkan have similar name than dx12? 
This feature is in RHI level. I think I will need to update comment to remove the "undefined swizzle" since it could be Dx12 implementation specific.

Reply:
Does Metal and Vulkan have similar name than dx12?
This feature is in RHI level. I think I will need to update comment to remove the "undefined swizzle" since it could be Dx12 implementation specific.

Reply:
I don't believe this concept exists in the other RHIs but I could be wrong. If it's DX12 specific though, it wouldn't be the comment we'd want to remove, we'd want to move this variable internally within the DX12 RHI (since changing the comment doesn't change what the variable means)